ods_ci/libs/DataSciencePipelinesAPI.py

```diff
@@ -15,14 +15,6 @@ def __init__(self):
         self.route = ''
         self.sa_token = None
 
-    @keyword
-    def create_operator(self, file):
-        print(os.path.abspath(__file__))
-        current_path = __file__.index('/libs/')
-        current_path = __file__[:current_path]
-        response, error = self.run_oc(f"oc apply -f {current_path}/{file}")
-        print(f'create_operator({error}): {response}')
-
     @keyword
     def get_redhat_openshift_pipelines(self):
```

Comment:
In my opinion this method should be called `Wait Until OpenShift Pipelines Operator Is Deployed` and be added as keyword in ods_ci/tests/Resources/Operators/OpenShiftPipelines  (new file to have keywords related to this operator, see comment below)
	  
Alternativelly, we already have some keywords handling installation of operators in ods_ci/tests/Resources/Page/OCPDashboard/OperatorHub/OperatorHub.robot and ods_ci/tests/Resources/Page/OCPDashboard/InstalledOperators/InstalledOperators.robot